utils/qrtracking.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+from functools import wraps
+
+from reconcile.status import RunningState
+from utils.metrics import func_runtime
+
+
+def track(function):
```

Comment:
What about? Maybe too long (again)
```suggestion
def elapsed_seconds_from_commit_metric(function):
```

Reply:
That's terrible, but I like it :)